.lando.base.yml

```diff
@@ -1,5 +1,4 @@
 name: drupal-project
-recipe: drupal10
```

Comment:
There's not much point in having recipe in .lando.base.yml if we want developers to keep it in their local .lando.yml